hydra-node/src/Hydra/Chain/Direct/Util.hs

```diff
@@ -190,3 +196,64 @@ isMarkedOutput :: TxOut CtxUTxO -> Bool
 isMarkedOutput = \case
   (TxOut _ _ (TxOutDatumHash ha) _) -> ha == markerDatumHash
   _ -> False
+
+-----------------------------------------------------------------------------
+-- TEST RELATED FUNCTIONS
```

Comment:
I would put these functions in the test module if they are test related.

Reply:
I reverted this change as it is no longer needed.